client/src/components/form/Dropdown/Dropdown.jsx

```diff
@@ -6,6 +6,12 @@ import { useWrapperRef } from '../../../hooks/useWrapperRef';
 import Arrow from '../../../../assets/icons/angle-down-solid.svg';
 
 const Dropdown = ({ values, onSelect, label, isDisabled, initialSelectedIndex }) => {
+  useEffect(() => {
+    if (initialSelectedIndex !== undefined) {
+      onSelect(values[initialSelectedIndex]);
```

Comment:
Is there any case where this index could be `undefined`?

Reply:
Yes! Usually it will be defined but if its undefined it wont break things!